test/dispersion.jl

```diff
@@ -0,0 +1,83 @@
+using Entropies, Test
+
+@testset "Dispersion methods" begin
+    x = rand(100)
+
+    @testset "Dispersion" begin
+
+        @testset "Internals" begin
```

Comment:
The property **robust** of a good test suite means that one tests that the expected functionality is met, not how it is met. Subsequently, only exported API should be used in the test suite. The internals are not tested, their final outcome is. See also https://github.com/JuliaDynamics/Entropies.jl/issues/85